client/components/swipeable/index.js

```diff
@@ -0,0 +1,249 @@
+import classnames from 'classnames';
+import { useRtl } from 'i18n-calypso';
+import { Children, useState, useEffect, useRef } from 'react';
+
+import './style.scss';
+
+export const Swipeable = ( {
+	hasDynamicHeight = false,
+	children,
+	currentPage = 0,
+	onPageSelect,
+	pageClassName,
+	containerClassName,
+} ) => {
+	const [ pageWidth, setPageWidth ] = useState( null );
+	const [ swipeableArea, setSwipeableArea ] = useState();
+	const isRtl = useRtl();
+
+	const [ pagesStyle, setPagesStyle ] = useState( {
+		transform: `translate3d(0px, 0px, 0px)`,
+		transitionDuration: `300ms`,
+	} );
+	const [ dragStartData, setDragStartData ] = useState();
```

Comment:
With the recent changes, the initial value should also be `null` for consistency

```suggestion
	const [ dragStartData, setDragStartData ] = useState( null );
```